conda_smithy/feedstock_outputs.py

```diff
@@ -0,0 +1,250 @@
+"""
+This module registers and validates feedstock outputs.
+"""
+import tempfile
+import os
+import json
+import hmac
+import urllib.parse
+
+from binstar_client.utils import get_server_api
+from binstar_client import BinstarError
+
+import git
+
+from .feedstock_tokens import is_valid_feedstock_token
+
+
+def _get_ac_api():
+    """wrap this a function so we can more easily mock it when testing"""
+    from .ci_register import anaconda_token
+
+    return get_server_api(token=anaconda_token)
+
+
+def copy_feedstock_outputs(
+    outputs, staging_conda_channel, production_conda_channel
+):
+    """Copy outputs from one chanel to another.
+
+    Parameters
+    ----------
+    outputs : dict
+        A dictionary mapping each full qualified output in the conda index
+        to a hash ("md5"), its name ("name"), and version ("version").
```

Comment:
Why md5?

Reply:
This is what is in the anaconda.org API. We have to use that to validate that the right outputs are being copied. The feedstock CI jobs will send the hashes and then we check them against the anaconda API.

Reply:
Ahh OK!